HLTrigger/Configuration/python/HLT_75e33/modules/hltHpsSelectedPFTauLooseTauWPDeepTau_cfi.py

```diff
@@ -0,0 +1,14 @@
+import FWCore.ParameterSet.Config as cms
+
+hltHpsSelectedPFTauLooseTauWPDeepTau = cms.EDFilter("PFTauSelector",
+    cut = cms.string('pt > 27 && abs(eta) < 2.1'),
```

Comment:
for my education what's the reason to limit the cut to |eta|<2.1 (also in phase-2)?